cpp/src/arrow/flight/sql/odbc/flight_sql/accessors/common.h

```diff
@@ -43,7 +44,7 @@ inline size_t CopyFromArrayValuesToBinding(ARRAY_TYPE* array, ColumnBinding* bin
       }
     }
   } else {
-    // Duplicate this loop to avoid null checks within the loop.
+    // Duplicate above for loop to exit early when null value is found
```

Comment:
Nit: let's make this `for-loop` instead of `for loop`.

Reply:
fixed